### PR TITLE
added a basic flask template 

### PR DESCRIPTION
**Pull request by sirusb**
Added a basic template for an app that is supposed to predict the spread of COVID-19 in Algeria vs hospital capacity.

### CODE REVIEW

**Comment by Taybou** [comment]
Hello @sirusb, 

Thank you for bootstrapping the project, I suggest to remove all the static part and maybe we can do it with react, what did you think?

Cheers

File: DZCovidPred/app/predictions/views.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+from uuid import uuid4
+
+from flask import Blueprint, render_template, current_app, request, flash, \
+    url_for, redirect, session, abort, jsonify
+
+predictions = Blueprint("predictions",__name__, static_folder="../static", template_folder="../template")
+
+@predictions.route("/")
+def index():
+    if ('uid' not in session.keys() or session['uid'] is None or len(str(session['uid'])) == 0 ):
```
Comment (01walid) [inline DZCovidPred/app/predictions/views.py]: All that can be `if not session.get('uuid', False):`

**Comment by 01walid** [comment]
> Hello @sirusb,
> 
> Thank you for bootstrapping the project, I suggest to remove all the static part and maybe we can do it with react, what did you think?
> 
> Cheers

+1

**Comment by Taybou** [comment]
Question: I prefer to remove the parent folder DZCovidPred. Is it your idea to separate the project?

**Comment by sirusb** [comment]
@Taybou it just seems that there are many visualization apps, so each one will be in a folder, and there will be a main app that redirects to them all. What do you think?

File: DZCovidPred/app/predictions/views.py (hunk 1 above)
Comment (sirusb) [reply DZCovidPred/app/predictions/views.py]: Great! No problem.

**Comment by sirusb** [comment]
> Thank you for bootstrapping the project, I suggest to remove all the static part and maybe we can do it with react, what did you think?
Hi @Taybou yeah, sure! I am just not very familiar with REACT, but if someone can do it , it would be great.

**Comment by Taybou** [comment]
@sirusb all right!

IMHO; we should focus on one app for now, and later if we see the need to extend to other apps we will see then.

For "react", I can rely on that, and maybe we also have @01walid or involve other people too.

**Comment by sirusb** [comment]
@Taybou Sounds great! I will do the requested changes and add some functionality today afternoon (Boston time).

**Comment by 01walid** [comment]
@Taybou I merged for now, we'll enhance or separate backend/frontend over time.

**Pull request merged**